frontend/src/pages/cans/list/CanList.jsx

```diff
@@ -14,32 +14,38 @@ const CanList = () => {
 
     return (
         <>
-            <BreadcrumbList>
-                <BreadcrumbItem isCurrent pageName="CANs" />
-            </BreadcrumbList>
+            <header>
+                <BreadcrumbList>
+                    <BreadcrumbItem isCurrent pageName="CANs" />
+                </BreadcrumbList>
+            </header>
 
-            <table className="usa-table usa-table--borderless">
-                <caption>List of all CANs</caption>
-                <thead>
-                    <tr>
-                        <th scope="col">number</th>
-                        <th scope="col">description</th>
-                    </tr>
-                </thead>
-                <tbody>
-                    {canList.map((can) => (
-                        <tr key={can.id}>
-                            <th scope="row">
-                                <Link id="lnkCans" to={"./" + can.id}>
-                                    {can.number}
-                                </Link>
-                            </th>
-                            <td>{can.description}</td>
+            <nav>
+                <h1>CANs</h1>
+
+                <table className="usa-table usa-table--borderless">
+                    <caption>List of all CANs</caption>
+                    <thead>
+                        <tr>
+                            <th scope="col">number</th>
+                            <th scope="col">description</th>
                         </tr>
-                    ))}
-                </tbody>
-            </table>
-            <Outlet />
+                    </thead>
+                    <tbody>
+                        {canList.map((can) => (
+                            <tr key={can.id}>
+                                <th scope="row">
+                                    <Link to={"./" + can.id}>{can.number}</Link>
+                                </th>
+                                <td>{can.description}</td>
+                            </tr>
+                        ))}
+                    </tbody>
+                </table>
+            </nav>
+            <main>
```

Comment:
Shouldn't the `<main>` wrap around the `<nav>`?

Reply:
Hmmm, I'll look into that.

Reply:
Fixed.